frontend/lib/src/components/widgets/DataFrame/columns/LinkColumn.ts

```diff
@@ -104,35 +103,45 @@ function LinkColumn(props: BaseColumnProps): BaseColumn {
     sortMode: "default",
     validateInput,
     getCell(data?: any, validate?: boolean): GridCell {
+      if (!data) {
```

Comment:
I think this part isn't 100% correct since there is a small semantical difference between, and empty string and a missing value (value = null or undefined).